rep-2001.rst

```diff
@@ -634,6 +637,35 @@ Desktop
                  teleop_twist_keyboard, tlsf, tlsf_cpp,
                  topic_monitor, turtlesim]
 
+Perception
+""""""""""
+
+::
+
+  - perception:
+      extends:  [ros_base]
+      packages: [image_common, image_pipeline, image_transport_plugins,
+                 laser_filters, laser_geometry, perception_pcl,
+                 vision_opencv]
+
+Simulation
+""""""""""
+
+::
+
+  - simulation:
+      extends:  [ros_base]
```

Comment:
I'd actually suggest that the `simulation` variant extend `desktop`.  Presumably if a user is going to use simulation, they also want general debugging and other GUI tools.  But I don't feel strongly about it, so if you think `ros_base` is more appropriate I'm fine with it.

Reply:
I don't have a strong feeling either way, so I roughly followed what was done in ROS 1, where it extends `robot`.

Reply:
I would suggest keeping with the base. So that the simulation can be used in a more embedded type of environment, such as on a cloud machine. There will be some integrated unused gui elements but this sort of application. But the developer tools and introspection tools are not necessary. And that union of desktop + simulation is covered effectively desktop_full with only perception added in addition.

Reply:
> embedded type of environment

headless environment, probably the better way to say it. And yeah, we run lots of headless simulations in RoboMaker, for example. The gazebo frontend is never invoked in the actual simulation environment, and instead can be connected at will from another instance.

Reply:
So, `ros_ign` includes `ros_ign_gazebo_demos`, which [brings tools like RQt and RViz](https://github.com/ignitionrobotics/ros_ign/blob/8530ab1189d0738a2af438ff8d740681abf794ec/ros_ign_gazebo_demos/package.xml#L17-L25). If we're targeting a headless use case, I'd break this up a bit more and remove the demos from `simulation`, leaving them only on the `desktop-full` / `quickstart` variant, which builds on top of `desktop`.

Reply:
Removed `ros_ign_gazebo_demos` from the `simulation` variant in 86241d49b523304ce7d95f877f2e7cb38a5ace70.